src/pages/RepoPage/FlagsTab/subroute/FlagsTable/hooks/useRepoFlagsTable.ts

```diff
@@ -27,48 +32,48 @@ const createMeasurementVariables = (historicalTrend, oldestCommitAt) => {
   return { afterDate, interval }
 }
 
-function useRepoFlagsTable(isDesc) {
+type URLParams = {
+  provider: string
+  owner: string
+  repo: string
+}
+
+function useRepoFlagsTable(isDesc: boolean) {
   const { params } = useLocationParams({
     search: '',
-    historicalTrend: '',
+    historicalTrend: 'LAST_3_MONTHS',
     flags: [],
   })
-  const { provider, owner, repo } = useParams()
+  const { provider, owner, repo } = useParams<URLParams>()
   const { data: repoData } = useRepo({
     provider,
     owner,
     repo,
   })
   const isAdmin = repoData?.isAdmin
+  // @ts-expect-errors, useLocation params needs to be updated to have full types
   const isSearching = Boolean(params?.search)
-  const [sortBy, setSortBy] = useState(SortingDirection.ASC)
   const { afterDate, interval } = createMeasurementVariables(
-    params?.historicalTrend,
+    // @ts-expect-errors, useLocation params needs to be updated to have full types
+    params?.historicalTrend ?? 'LAST_3_MONTHS',
```

Comment:
Maybe we can use these constants instead of the magic strings for the time_option_values

https://github.com/codecov/gazebo/blob/7d02fc94a725b506fe3af81aebddaf851bbc1ede/src/pages/RepoPage/FlagsTab/constants.ts#L22-L28

Reply:
good call.